pycaret/containers/models/time_series.py

```diff
@@ -856,6 +856,59 @@ def _set_tune_distributions(self) -> Dict[str, List[Any]]:
             }
         return tune_distributions
 
+class CrostonContainer(TimeSeriesContainer):
+    """
+    SKtime documentation:
+    https://www.sktime.org/en/latest/api_reference/auto_generated/sktime.forecasting.croston.Croston.html
+
+    """
+    model_type = TSModelTypes.CLASSICAL
+
+    def __init__(self, globals_dict: dict) -> None:
+        logger = get_logger()
+        np.random.seed(globals_dict["seed"])
+        self.gpu_imported = False
+
+        from sktime.forecasting.croston import Croston  # type: ignore
+
+        dummy = Croston()
+        self.active:bool = self.disable_pred_int_enforcement(
+            forecaster=dummy, enforce_pi=globals_dict["enforce_pi"]
+        )
+        if not self.active:
+            return
+
+        self.strictly_positive = globals_dict.get("strictly_positive")
+
+        tune_grid = self._set_tune_grid
+        tune_distributions = self._set_tune_distributions
+        leftover_parameters_to_categorical_distributions(tune_grid, tune_distributions)
+
+        super().__init__(
+            id="croston",
+            name="Croston",
+            class_def=Croston,
+            tune_grid=tune_grid,
+            tune_distribution=tune_distributions,
+            is_gpu_enabled=self.gpu_imported
+        )
+
+    @property
+    def _set_tune_grid(self) -> Dict[str, List[Any]]:
+        # Lack of documentation here, SKtime and R implementations are default 0.1
+        # using : np.logspace(start=0.01,stop=1.0,endpoint=True, base=100, num=8)/100
+        smoothing_grid: List[float] = [0.01 , 0.02 , 0.039, 0.074, 0.142, 0.272, 0.521, 1.   ]
```

Comment:
Why such specific values? Should it not be generic like `0.01, 0.03, 0.1, 0.2. 0.4, 0.6, 0.8, 1.0`?

Reply:
In the line above I used a logspace to define these so that they roughly follow the probabilities in the UniformDistribution with log=True, happy to change back to the generic grid you suggested?

Reply:
OK, I think the alternate generic suggestions would be better. We are not looking for precise values, just that we sample roughly equally between 0.01-0.1 and between 0.1-1.0. You can see equivalent examples in the other models.

https://github.com/pycaret/pycaret/blob/3b1ff2268d28a536f2d96930f5d2eec06f772c29/pycaret/containers/models/regression.py#L1752

Reply:
Implemented as suggested